validator/schema/upsun.json

```
@@ -849,6 +858,10 @@
                     "start": {
                       "type": "string",
                       "title": "The command used to start the application.  It will be restarted if it terminates. Do not use on PHP unless using a custom persistent process like React PHP."
+                    },
+                    "post_start": {
+                      "type": "string",
+                      "title": "The command used to run after starting the application."
```

```suggestion
                      "title": "A command executed after the application is started."
```

(from the git source, and "used to run" seemed a bit strange)